src/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -385,3 +385,21 @@ function objective_functionize_bridge(b::LazyBridgeOptimizer)
 end
 
 bridging_cost(b::LazyBridgeOptimizer, args...) = bridging_cost(b.graph, node(b, args...))
+
+function MOI.compute_conflict!(model::LazyBridgeOptimizer)
+    return MOI.compute_conflict!(model.model)
+end
+
+function MOI.get(
+    model::LazyBridgeOptimizer,
+    ::MOI.ConstraintConflictStatus,
+    c::MOI.ConstraintIndex,
+)
+    if is_bridged(model, c)
+        error(
+            "Unable to query `ConstraintConflictStatus` of $(c) because the " *
```

Comment:
See https://github.com/jump-dev/MathOptInterface.jl/issues/1203#issuecomment-744303302

Reply:
It seemed better to throw a nicer error message than:
```Julia
julia> MOI.get(model, MOI.ConstraintConflictStatus(), d)
ERROR: ArgumentError: Bridge of type `MathOptInterface.Bridges.Constraint.ScalarSlackBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}` does not support accessing the attribute `MathOptInterface.ConstraintConflictStatus()`.
Stacktrace:
 [1] get(::MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer}, ::MathOptInterface.ConstraintConflictStatus, ::MathOptInterface.Bridges.Constraint.ScalarSlackBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge.jl:61
 [2] (::MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus})(::MathOptInterface.Bridges.Constraint.ScalarSlackBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:920
 [3] (::MathOptInterface.Bridges.var"#1#2"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}},MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus}})() at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:242
 [4] call_in_context(::MathOptInterface.Bridges.Variable.Map, ::Int64, ::MathOptInterface.Bridges.var"#1#2"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}},MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/Variable/map.jl:404
 [5] call_in_context(::MathOptInterface.Bridges.Variable.Map, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}, ::Function) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/Variable/map.jl:435
 [6] call_in_context(::MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer}, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}, ::MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:241
 [7] get(::MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer}, ::MathOptInterface.ConstraintConflictStatus, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:920
 [8] get(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::MathOptInterface.ConstraintConflictStatus, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Utilities/cachingoptimizer.jl:605
 [9] _moi_get_result(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::MathOptInterface.ConstraintConflictStatus, ::Vararg{Any,N} where N) at /Users/oscar/.julia/packages/JuMP/qhoVb/src/JuMP.jl:848
 [10] get(::Model, ::MathOptInterface.ConstraintConflictStatus, ::ConstraintRef{Model,MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}},ScalarShape}) at /Users/oscar/.julia/packages/JuMP/qhoVb/src/JuMP.jl:887
 [11] top-level scope at REPL[19]:1
```

The default error will just cause people to file issues.

Reply:
We could say that for any attribute though, how is this one specific ? It also does not seem unreasonable to implement it for several bridges so the error message should mention that the issue is that a specific bridge type does not implement it and that's exactly what the error message is saying here. Maybe we could improve the error message for all attribute then if you feel it's not clear.

Reply:
I've removed the specific error. We can either improve the error message or implement `ConstraintConflictStatus` for bridges.

Reply:
Opened an issue to track: https://github.com/jump-dev/MathOptInterface.jl/issues/1221